Objects/bytearrayobject.c

```diff
@@ -2391,11 +2391,17 @@ PyDoc_STRVAR(length_hint_doc,
 static PyObject *
 bytearrayiter_reduce(bytesiterobject *it, PyObject *Py_UNUSED(ignored))
 {
+    PyObject *iter = _PyEval_GetBuiltin(&_Py_ID(iter));
+
+    /* _PyEval_GetBuiltin can invoke arbitrary code.
+     * calls must be *before* access of `it` pointers,
+     * since C/C++ parameter eval order is undefined.
```

Comment:
```suggestion
     * since C parameter eval order is undefined.
```
This is C code, so we don't care about C++.

Reply:
removed C++ references in 
[45522c6](https://github.com/python/cpython/pull/101769/commits/45522c6dd8f6d39b3aeb05015371e31a88168d29)